client/src/components/Explorer/actions.test.js

```diff
@@ -65,32 +65,22 @@ describe('actions', () => {
     });
   });
 
-  describe('popPage', () => {
+  describe('gotoPage', () => {
     it('exists', () => {
-      expect(actions.popPage).toBeDefined();
-    });
-
-    it('works', () => {
-      expect(actions.popPage().type).toEqual('POP_PAGE');
-    });
-  });
-
-  describe('pushPage', () => {
-    it('exists', () => {
-      expect(actions.pushPage).toBeDefined();
+      expect(actions.gotoPage).toBeDefined();
     });
 
     it('creates action', () => {
       const store = mockStore(stubState);
-      store.dispatch(actions.pushPage(5));
+      store.dispatch(actions.gotoPage(5));
```

Comment:
Here and below – it looks like `gotoPage`’s transition argument needs to be defined?